lib/FrameManager.js

```diff
@@ -276,7 +276,7 @@ class FrameManager extends EventEmitter {
       frameId: frame._id,
       grantUniveralAccess: true,
       worldName: name,
-    })));
+    }).catch(debugError)));
```

Comment:
comment why this can fail.  Frames die while we are creating worlds in them?

Reply:
Yes, that's the pattern we already use in code.